### PR TITLE
Add info how to disable authentication

### PR DESCRIPTION
**Pull request by spacemanspiff2007**
Signed-off-by: Sebastian Gerber <github@sgerber.com>

### CODE REVIEW

File: configuration/restdocs.md [hunk 1]
```diff
@@ -87,6 +87,18 @@ To authenticate with an API token, **apaddpend `-u '{API_TOKEN}:'` to the comman
 
 You can manage all access tokens in your profile settings in the Main UI.
 
+### Disable authentication
+
+It is possible to disable authentication.
+Stop openhab and add this part to ```userdata/etc/org.apache.karaf.features.xml``` before ```</featuresProcessing>```:
```
Comment (Confectrician) [inline configuration/restdocs.md:93]: We should consider using `$OPENHAB_USERDATA` as term placeholder here like done in other articles.

```
$OPENHAB_USERDATA/etc/org.apache.karaf.features.xml
```

**Review by Confectrician — COMMENTED** [review]
Hi @spacemanspiff2007 and thanks for the contribution,

Alltough i would call this an advanced rest topic, i have no better place in mind for this part.
So we can leave it here for now. 

You sign-off, alltough added correct, seems to have issues.
Do have conencted this email adress with github already?
At least it is not public available from your profile.

Just to be clear:
That does **not** affect your PR, since the sign-off is ok from my side and i will override the check in case its needed.
I am just curious, to understand the reason why check is failing here to learn for the future.

**Comment by spacemanspiff2007** [comment]
> You sign-off, alltough added correct, seems to have issues.
> Do have conencted this email adress with github already?
> At least it is not public available from your profile.

It's because of the github privacy feature that strips mail addresses from commits.
I have to manually set the mail address to override it but because I created this commit from the web gui I am not able to.
The only way is to create the PR locally and then push it to Github.

I have made the changes as you requests. :+1:

**Comment by Confectrician** [comment]
But it should have worked for your first commit was correctly signed off and the DCO check failed too.

It is missing for the second commit, that's correct.
(You can simply add it in the commit message yourself and it will already work.)

For the first commit, GitHub expected something different than you have send.

image

I would guess that this is related to the profile settings.

**Review by Confectrician — APPROVED** [review]
Thanks 👍

**Pull request merged**